contributions/covid19/tommysam-jii/COVID19_COURSE_INFO.md

```diff
@@ -0,0 +1,104 @@
+* The course information cannot be longer than 60 characters.
+
+If you don't fulfill these criteria, you will receive an error message with the information you need to add or change to be able to successfully make a submission.
+
+<a name="dev"> </a>
+
+### Devops Tools
+
+#### Jenkins 
+One of the intended learning outcomes for the course Automated Software Testing and DevOps DD2482, is that the student should be able to setup and deploy continuous integration. Therefore, we decided to try to create our own Jenkins server and integrate it with our COVID-19 project. 
+
+We are running Jenkins as Docker container from the Blue Ocean Docker image. We have created two different pipelines. One is the main pipeline that are connected to the original GitHub repository for our project. The other one is a pipeline connected to the a forked copy of the original repository, which we use to test changes in the set up of the pipeline, or when we try to build new stages for the original pipeline.
+
+Right now, we only have a Build stage in our main pipeline. Our Build stage install all necessary dependencies and package for our React application. If we continue to work with this project after the course has ended, we also want to create a Test stage and a Deliver stage. We have tried to fix a Deliver stage in the pipeline for our forked repository, but with no further success. If there are someone who has previous knowledge or experience with creating Jenkins' pipelines for a React/NodeJS application, we would like some help with our issue [#528](https://github.com/KTH/devops-course/issues/528).
+
+Today, our Jenkins server is running locally on Johanna's computer. To get access to our Jenkins server, you need to go to [92.34.7.239:8080](92.34.7.239:8080) and log in with these credentials:
+
+Username: test_user  
+Password: Test2482
+
+The server might not be up and running all the time, but down below we have a screencast of our Jenkins server in action. If the server is not up and running and you want to test our pipeline, please contact Johanna at jii@kth.se.
+
+Click on the image below to be re-directed to the screencast at Youtube:
+
+[![](/images/jenkins_screencast.png)](https://youtu.be/XFCERzBMqZc) 
+
+The video has been cut at some points to shorten the length of the video and remove parts that does not add any new information to the filmed screencast. 
+
+#### Firebase
+Firebase is a serverless computing service, which provides a real-time database for it's users. Firebase provides an API that allows data to be synchronized across all the clients. The application data is stored on Firebase's own cloud.[[1]](https://en.wikipedia.org/wiki/Firebase) Serverless computing is relevant for DevOps, since it shortens the release cycle. By using a serverless architecture, the developers can focus on the functionality of the application and the application's GUI, instead of writing scripts for the server or the database. 
```

Comment:
Add link to Firebase config and client code in your app. Discuss the code and its simplicity/complexity.

Reply:
What do you mean by adding the Firebase config? Do you want us to add you as a collaborator to our project in Firebase?

Reply:
> Do you want us to add you as a collaborator to our project in Firebase?

no

> What do you mean by adding the Firebase config?

To explain the configurations in the public repo.